modules/data-streams/src/internalClusterTest/java/org/elasticsearch/datastreams/DataStreamsSnapshotsIT.java

```diff
@@ -194,50 +220,63 @@ public void testSnapshotAndRestore() throws Exception {
         );
     }
 
-    public void testSnapshotAndRestoreAllDataStreamsInPlace() throws Exception {
+    public void testFailureStoreSnapshotAndRestore() throws Exception {
```

Comment:
Just checking: Was this meant to be a new test case or did you mean to replace the previous test case with this?

Reply:
Weird, it's been a while, I would assume now that I would want to add a test case. I will dive in. Thanks for pointing it out.